scripts/plugins/storage/sqlite/init.sql

```diff
@@ -600,6 +600,23 @@ CREATE TABLE fledge.filter_users (
              name        character varying(255)        NOT NULL,
              user        character varying(255)        NOT NULL);
 
+-- Create control_script table
+-- Script management for control dispatch service
+CREATE TABLE fledge.control_script (
+             name          character varying(255)        NOT NULL,
+             steps         JSON                          NOT NULL DEFAULT '{}',
+             acl           character varying(255),
+             CONSTRAINT    control_script_pkey           PRIMARY KEY (name) );
+
+-- Create control_acl table
+-- Access Control List Management for control dispatch service
+CREATE TABLE fledge.control_acl (
+             name          character varying(255)        NOT NULL,
+             service       character varying(255),
```

Comment:
The service needs to be a JSON elements as well. See https://wiki.lfedge.org/display/FLEDGE/Adding+Control+to+Fledge

Reply:
Done. Thanks